src/Sepes.Common/Dto/UserDto.cs

```diff
@@ -26,7 +30,7 @@ public UserDto()
           
         }
 
-        public UserDto(string objectId, string userName, string fullName, string email, bool admin = false, bool sponsor = false, bool datasetAdmin = false, bool employee = false)
+        public UserDto(string objectId, string userName, string fullName, string email, bool admin = false, bool sponsor = false, bool datasetAdmin = false, bool employee = false, bool external = false)
```

Comment:
Could this be an object instead as a parameter?

Reply:
Ate some point one would have to instantiate a class, so we would only bite our self in the foot